local_setup/README.md

```diff
@@ -0,0 +1,16 @@
+help: `go run sha_generator.go -h ` or `go run sha_generator.go -help`
+
+Please see script's help for usage explanation and arguments' default values. All arguments can be combined.
+
+- generate one SHA and print on console (default behavior) - `go run sha_generator.go`
+- generate <NUM> SHAs and print on console - `go run sha_generator.go -num NUM`
+
+- specify which org ID and account - `go run sha_generator.go -org 12345 -account 12345678`
+  - This will create one SHA-256 as `-num` is not specified.
+  - `org` and `account` do not influence the result as it is not sent to Kafka nor stored in DB.
+
+- send generated data to Kafka (see help for related configuration) - `go run sha_generator.go -produce`
+- store  generated data in DB (see help for related configuration) - `go run sha_generator.go -store`
+
+- generate <NUM> SHAs and send to kafka - `go run sha_generator.go -num NUM -produce`
```

Comment:
I was not able to connect to kafka in docker on port 9092 from localhost. Kafka seems to be verifying the hostname somehow and doesn't like I'm connecting to URL `localhost:9092` and not `kafka:9092`. After a bit googling I found a solution - setting up Kafka to have second listener for localhost (it has to be on different port), I had to set this in kafka.env:

```
KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=PLAINTEXT:PLAINTEXT,PLAINTEXT_EXTERNAL:PLAINTEXT
KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,PLAINTEXT_EXTERNAL://:9093
KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092,PLAINTEXT_EXTERNAL://localhost:9093
```

After this and changing the port to 9093 in docker-compose.yml and sha_generator, `-produce` is working.